data/secscan_model/secscan_v4_model.py

```diff
@@ -56,6 +57,7 @@
 
 
 DEFAULT_SECURITY_SCANNER_V4_REINDEX_THRESHOLD = 86400  # 1 day
+TAG_LIMIT = 100
```

Comment:
Why the limit? What subset is chosen when it's exceeded?

Reply:
From what I understand the limit is for retrieving the tag names that point to that manifest up to that limit. So at max there will only be 100 tags attached to the event data for the notification.

Reply:
Right, so how is that 100 chosen?
The limit will have to be documented somewhere, also.

Alternatively, is it possible to page through all the tags and emit multiple events if needed?

Reply:
The security notification worker already sets it as 100 which is why I chose that number. It's also possible to set it as None for no limit. 
Is it better to just go through all tag names and not limit it to 100?

Reply:
Maybe? It could make the event object very large, which is why I suggested paging.